src/types/boolean.ts

```diff
@@ -16,9 +15,9 @@ export const types = {
 	},
 };
 
-export const fetchProcessing = Promise.method((data: any) => data === 1);
+export const fetchProcessing = async (data: any) => data === 1;
```

Comment:
Is there a benefit that all `fetchProcessing` are async?
Since afaict none of those defined in this repo is doing any async work.